src/API/Helpers/LocalizationHelper.cs

```diff
@@ -10,28 +10,37 @@ namespace SerousCommonLib.API.Helpers {
-		internal static readonly MethodInfo LocalizationLoader_SetLocalizedText = typeof(LocalizationLoader).GetMethod("SetLocalizedText", BindingFlags.NonPublic | BindingFlags.Static);
-		internal static readonly FieldInfo LanguageManager__localizedTexts = typeof(LanguageManager).GetField("_localizedTexts", BindingFlags.NonPublic | BindingFlags.Instance);
-
+		private static readonly MethodInfo LocalizationLoader_LoadTranslations = typeof(LocalizationLoader).GetMethod("LoadTranslations", BindingFlags.NonPublic | BindingFlags.Static);
+		private static readonly MethodInfo LocalizedText_SetValue = typeof(LocalizedText).GetMethod("SetValue", BindingFlags.NonPublic | BindingFlags.Instance);
+		
 		/// <summary>
 		/// Forces the localization for the given mod, <paramref name="mod"/>, to be loaded for use with <seealso cref="Language"/>
 		/// </summary>
 		/// <param name="mod">The mod instance</param>
-		public static void ForceLoadModHJsonLocalization(Mod mod) {
-			Dictionary<string, ModTranslation> modTranslationDictionary = new();
+#if TML_2022_09
+        public static void ForceLoadModHJsonLocalization(Mod mod) {
+            Dictionary<string, ModTranslation> modTranslationDictionary = new();
 
-			LocalizationLoader_AutoloadTranslations.Invoke(null, new object[] { mod, modTranslationDictionary });
+            LocalizationLoader_AutoloadTranslations.Invoke(null, new object[] { mod, modTranslationDictionary });
 
-			Dictionary<string, LocalizedText> dict = LanguageManager__localizedTexts.GetValue(LanguageManager.Instance) as Dictionary<string, LocalizedText>;
+            Dictionary<string, LocalizedText> dict = LanguageManager__localizedTexts.GetValue(LanguageManager.Instance) as Dictionary<string, LocalizedText>;
 
-			var culture = Language.ActiveCulture;
-			foreach (ModTranslation translation in modTranslationDictionary.Values) {
-				//LocalizedText text = new LocalizedText(translation.Key, translation.GetTranslation(culture));
-				LocalizedText text = Activator.CreateInstance(typeof(LocalizedText), BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.CreateInstance, null, new object[] { translation.Key, translation.GetTranslation(culture) }, CultureInfo.InvariantCulture) as LocalizedText;
+            var culture = Language.ActiveCulture;
+            foreach (ModTranslation translation in modTranslationDictionary.Values) {
+                //LocalizedText text = new LocalizedText(translation.Key, translation.GetTranslation(culture));
+                LocalizedText text = Activator.CreateInstance(typeof(LocalizedText), BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.CreateInstance, null, new object[] { translation.Key, translation.GetTranslation(culture) }, CultureInfo.InvariantCulture) as LocalizedText;
 
-				LocalizationLoader_SetLocalizedText.Invoke(null, new object[] { dict, text });
+                LocalizationLoader_SetLocalizedText.Invoke(null, new object[] { dict, text });
+            }
+        }
```

Comment:
needs tabs instead of spaces

Reply:
Added an editor config with default values from https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/code-style-rule-options

It does appear they want spaces by default, hence why my editor formatted it that way. The only edit was indent_style

Reply:
adding an `.editorconfig` is a bit overkill.  just replace the space indents with tabs

Reply:
Without an editor config, anyone that opens VS or Rider would need to change their project settings to use tabs instead of spaces.
Whereas an editor config tells the editor what styling to use.

I really think it would be good to have. The extra defaults could be removed however.

Reply:
I've removed the defaults. Some people prefer the defaults not to be there but personally I like them so I know what I can change without looking them up.

If you still feel strongly about removing it entirely though I will fold.

Reply:
that is fine, just make sure to change the indentation in the code

Reply:
Should be set, looks like that one line was just related to it being on a new line.

Reply:
lines 29 to 45 are still using spaces instead of tabs

Reply:
I am using VS auto formatter and that is what it's giving me. This would be the same case if you did it I would imagine. Forcefully entering the tabs would just cause conflicts every single time someone auto formatted this file.